config/boot.rb

```diff
@@ -1,4 +1,3 @@
-# Set up gems listed in the Gemfile.
 ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../Gemfile', __FILE__)
 
-require 'bundler/setup' if File.exist?(ENV['BUNDLE_GEMFILE'])
+require 'bundler/setup' # Set up gems listed in the Gemfile.
```

Comment:
Yes. Don't know the real reason, but AFAIK "require 'bundler/setup' "should always run in the initialization process. If the Gemfile is not found the app won't boot up any way.